mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -66,43 +73,48 @@ data class Widget(
     val rawInputHint: InputTypeHint?
 ) : Parcelable {
     val label get() = rawLabel.split("[", "]")[0].trim()
-    val stateFromLabel: String? get() {
```

Comment:
It would be good to get rid of the somewhat random reformatting here and below.

Reply:
This was done by Android Studio's default Kotlin formatting. Any tips what I should do in regards to this?

Reply:
We're probably missing some settings file in the repository. Did this reformatting happen 'magically' or did you do something to trigger it?

Reply:
I am going to confess here. I edit the files using VSCode, then go to Android Studio to reformat it using Cmd+Option+L then build / test it. One of these days I need to learn how to use Android Studio properly!

Reply:
I've manually restored the random formatting changes for now, but I'd love to get the automatic formatting done properly without messing up the established formatting.

Reply:
Can you check this setting and make sure it's set to "Project"? 

![grafik](https://github.com/openhab/openhab-android/assets/22525368/68f005d3-8aea-4a10-800c-6243429ba32c)

Reply:
Mine is also set to Project, but it's still adding those extra lines breaks

Reply:
<img width="980" alt="image" src="https://github.com/openhab/openhab-android/assets/2554958/8d67c6f2-c288-4567-91fb-a52a993f54f3">
<img width="683" alt="image" src="https://github.com/openhab/openhab-android/assets/2554958/588405a5-8f9b-4447-ab79-3532f335542f">
<img width="374" alt="image" src="https://github.com/openhab/openhab-android/assets/2554958/b288809b-520d-4d98-8a18-4da53c03601f">
<img width="492" alt="image" src="https://github.com/openhab/openhab-android/assets/2554958/5bccd797-6bc1-41f4-b0bf-90cab5a8a6e2">

Reply:
I'm still fighting with the linter. I tried running `ktlint --format` but it changed some existing code too.